packages/blade/src/components/Icons/Icons.stories.tsx

```diff
@@ -7,12 +7,62 @@ import {
   Stories,
   PRIMARY_STORY,
   Description,
-  Props,
+  ArgsTable,
```

Comment:
Had to replace this with `ArgsTable` since `Props` is deprecated and renamed to `ArgsTable` and will be removed in future versions. 
<img width="619" alt="Screenshot 2022-05-12 at 3 18 26 PM" src="https://user-images.githubusercontent.com/11384858/168047552-81456396-76c6-4e88-bac2-bd7a6cda8ca3.png">


// @chaitanyadeorukhkar

Reply:
Great, now we know 😄